integration-tests/tests/login_test.go

```diff
@@ -68,7 +68,7 @@ func (s *loginSuite) TestInvalidCredentialsError(c *check.C) {
 	err := s.writeCredentials(invalidLoginName)
 	c.Assert(err, check.IsNil, check.Commentf("error writting credentials"))
 
-	expectedMsg := "invalid credentials"
+	expectedMsg := "Provided email/password is not correct"
```

Comment:
With this expected message the test is failing for me, `error: access denied` works fine